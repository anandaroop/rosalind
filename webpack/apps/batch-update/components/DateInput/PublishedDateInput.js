import React from 'react'
import DateInput from './DateInput'

const PublishedDateInput = (props) => {
  return (
    <div>
      <DateInput
        selectDate={props.onSelectPublishedDate}
        placeholder='Select a published-after date'
      />
    </div>
  )
}

export default PublishedDateInput
