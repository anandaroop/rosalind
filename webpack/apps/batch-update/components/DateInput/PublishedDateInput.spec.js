import React from 'react'
import renderer from 'react-test-renderer'
import PublishedDateInput from './PublishedDateInput'

it('renders correctly', () => {
  const rendered = renderer.create(<PublishedDateInput />)
  const tree = rendered.toJSON()
  expect(tree).toMatchSnapshot()
})
