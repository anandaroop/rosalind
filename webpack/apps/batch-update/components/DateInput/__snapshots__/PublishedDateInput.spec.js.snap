exports[`test renders correctly 1`] = `
<div>
  <div>
    <input
      onChange={[Function]}
      placeholder="Select a published date"
      type="text"
      value="" />
    <div
      className="parsed">
      <a
        href="#"
        onClick={[Function]} />
    </div>
  </div>
</div>
`;
